plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -17,9 +17,15 @@
 import * as yup from 'yup';
 
 export type Component = {
-  id: string;
+  name: string;
 };
 
+export const componentShape: yup.Schema<Component> = yup
```

Comment:
Don't really think this is where we will end up defining the schema of components 😅. Thinking we pull in a JSONSchema blob eventually, but good to have some validation in place.